metagpt/utils/token_counter.py

```diff
@@ -34,9 +34,16 @@
     "glm-3-turbo": {"prompt": 0.0, "completion": 0.0007},  # 128k version, prompt + completion tokens=0.005￥/k-tokens
     "glm-4": {"prompt": 0.0, "completion": 0.014},  # 128k version, prompt + completion tokens=0.1￥/k-tokens
     "gemini-pro": {"prompt": 0.00025, "completion": 0.0005},
+    # Azure
+    "azure-gpt-3.5-turbo-4k": {"prompt": 0.0015, "completion": 0.002},
+    "azure-gpt-3.5-turbo-16k": {"prompt": 0.003, "completion": 0.004},
+    "azure-gpt-3.5-turbo-1106": {"prompt": 0.001, "completion": 0.002},
+    "azure-gpt-4-turbo": {"prompt": 0.01, "completion": 0.03},
+    "azure-gpt-4-turbo-vision": {"prompt": 0.01, "completion": 0.03},
+    "azure-gpt-4-8k": {"prompt": 0.03, "completion": 0.06},
+    "azure-gpt-4-32k": {"prompt": 0.06, "completion": 0.12},
```

Comment:
I don't think this code should be added. If it uses the same price system as OpenAI, then we shouldn't maintain two sets of code

Reply:
Considering that different service providers do not offer different quotes for the same service, consolidate the price lists from all service providers.